build.sbt

```diff
@@ -55,6 +60,8 @@ val timeBenchmark =
     .enablePlugins(JmhPlugin)
     .settings(commonSettings: _*)
     .settings(
+      // FIXME: does not compile with dotty
+      crossScalaVersions := List(scalaVersion.value),
```

Comment:
Various java parsing errors:
```scala
[error] -- [E006] Unbound Identifier Error: /home/smarter/opt/collection-strawman/benchmarks/time/src/main/java/streams/Java8Streams.java:19:14 
[error] 19 |    public int size;
[error]    |              ^
[error]    |              not found: ???
```
And this:
```scala
[error] -- Error: /home/smarter/opt/collection-strawman/benchmarks/time/src/main/scala/strawman/collection/immutable/ScalaTreeSetBenchmark.scala:60:62 
[error] 60 |  def map(bh: Blackhole): Unit = bh.consume(xs.map(x => x + 1))
[error]    |                                                              ^
[error]    |ambiguous implicits: both method newCanBuildFrom in object SortedSet and method newCanBuildFrom in class SortedSetFactory match type scala.collection.generic.CanBuildFrom[
[error]    |  scala.collection.immutable.TreeSet[scala.Long]
[error]    |, scala.Long, That] of parameter bf of method map in trait SetLike
[error] -- [E050] Reference Error: /home/smarter/opt/collection-strawman/benchmarks/time/src/main/scala/strawman/collection/immutable/ScalaTreeSetBenchmark.scala:60:36 
[error] 60 |  def map(bh: Blackhole): Unit = bh.consume(xs.map(x => x + 1))
[error]    |                                 ^^^^^^^^^^
[error]    |Ambiguous overload. The overloaded alternatives of method consume in class Blackhole with types
[error]    | (x$0: scala.Double): scala.Unit
[error]    | (x$0: scala.Float): scala.Unit
[error]    | (x$0: scala.Long): scala.Unit
[error]    | (x$0: scala.Int): scala.Unit
[error]    | (x$0: scala.Short): scala.Unit
[error]    | (x$0: scala.Char): scala.Unit
[error]    | (x$0: scala.Boolean): scala.Unit
[error]    | (x$0: scala.Byte): scala.Unit
[error]    | (x$0: scala.Any): scala.Unit
[error]    |all match arguments (That)
```

I haven't had time to minimize them and open bug reports against dotty unfortunately